Library/Formula/logstash.rb

```diff
@@ -2,19 +2,20 @@
 
 class Logstash < Formula
   homepage 'http://logstash.net/'
```

Comment:
Agree with the above. Since it's automatically redirecting, we should change it.